tests/integration/pages/page.dummy.js

```diff
@@ -0,0 +1,20 @@
+/*!
+ * @author:    Divio AG
+ * @copyright: http://www.divio.ch
+ */
+
+// ACCEPTANCE TEST
+//#############################################################################
+'use strict';
+/* jshint browser: true */
+/* jshint shadow: true */
+
+/**
+ * This file uses the Page Object pattern to define a test.
+ **/
+
+var DummyPage = {
```

Comment:
> Name variablesLikeThis, ClassesLikeThis, CONSTANTS_LIKE_THIS and events-like-this.

from http://aldryn-boilerplate-bootstrap3.readthedocs.org/en/latest/guidelines/javascript.html#naming
doesn't look like a class to me, so nope